shadow.c

```diff
@@ -117,10 +118,19 @@ static PHP_GSHUTDOWN_FUNCTION(shadow)
 }
 /* }}} */
 
+/* {{{ shadow_deps
+ */
+static const zend_module_dep shadow_deps[] = {
+	ZEND_MOD_OPTIONAL("zip")
```

Comment:
Unfortunately I don't see other way of fixing travis failure.
When tests are run the following code is executed:
`/.../bin/php  -n -c '/.../tmp-php.ini' ... -d "extension=shadow.so" -d "extension=/usr/lib64/php/modules/zip.so"  -d "shadow.override=ziparchive::open@w0,ziparchive::addfile@0" -f "/.../tests/override_class.php"`
This line is generated for `override_class` test by php test framework and we have no control over it - https://github.com/php/php-src/blob/PHP-5.4.45/run-tests.php. We can guarantee extension load order only by specifying it in the extension itself.